src/ui/actions/GetTaskToShowLabelsAction.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by Anton Ovcharenko on 18.01.2022.
+//
+
+#ifndef TASKMANAGER_SRC_UI_ACTIONS_GETTASKTOSHOWLABELSACTION_H_
+#define TASKMANAGER_SRC_UI_ACTIONS_GETTASKTOSHOWLABELSACTION_H_
+
+#include "Action.h"
+
+class GetTaskToShowLabelsAction : public Action {
+public:
+    explicit GetTaskToShowLabelsAction(const std::string &arg);
+
+public:
+    ActionResult execute(const std::shared_ptr<ModelInterface> &) override;
+
+private:
+    std::string arg_;
```

Comment:
better to give TaskId to Actions, not human readable strings

Reply:
will be done in a refactoring PR